lib/expand.js

```diff
@@ -63,7 +63,7 @@ module.exports = api;
  *
  * @return a Promise that resolves to the expanded value.
  */
-api.expand = ({
+api.expand = async ({
```

Comment:
looks like another important change for the changelog.

Reply:
See above. These are internal functions that were previously synchronous. The external ones were already asynchronous. That being said, there *is* a separate change that removes callback support on the public API, but the CHANGELOG covers that already.